Packs/Reco/Integrations/Reco/README.md

```diff
@@ -0,0 +1,21 @@
+Reco is a Saas security solution that protects your data from accidental leaks and malicious attacks.
```

Comment:
❯ markdownlint README.md
README.md:1:81 MD013/line-length Line length [Expected: 80; Actual: 101]
README.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "Reco is a Saas security soluti..."]
README.md:2 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above] [Context: "## Configure Reco on Cortex XSOAR"]
README.md:8:81 MD013/line-length Line length [Expected: 80; Actual: 113]
README.md:9:81 MD013/line-length Line length [Expected: 80; Actual: 94]
README.md:10:81 MD013/line-length Line length [Expected: 80; Actual: 100]
README.md:11:81 MD013/line-length Line length [Expected: 80; Actual: 98]
README.md:12:81 MD013/line-length Line length [Expected: 80; Actual: 104]
README.md:13:81 MD013/line-length Line length [Expected: 80; Actual: 105]
README.md:14:81 MD013/line-length Line length [Expected: 80; Actual: 100]
README.md:15:81 MD013/line-length Line length [Expected: 80; Actual: 102]
README.md:16:81 MD013/line-length Line length [Expected: 80; Actual: 101]
README.md:17:81 MD013/line-length Line length [Expected: 80; Actual: 98]
README.md:21 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
README.md:22 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 3]